docs/src/main/asciidoc/_elytron/components/Aggregate_Security_Realm.adoc

```diff
@@ -33,6 +35,16 @@ The following command is exactly the same except the `authorization-realms` attr
     authorization-realms=[jdbc-realm])
 ----
 
+The following command will define a transformer to be used to transform the principal after authentication.
+
+[source,options="nowrap"]
+----
+/subsystem=elytron/aggregate-realm=combined:add(
+    authentication-realm=properties-realm,
+    authorization-realms=[jdbc-realm],
+    principal-transformer=custom-transformer)
```

Comment:
May be good to mention that the configured principal transformer is a reference to a ```principal-transformer``` that has been defined in the ```mappers``` configuration.

Reply:
Added